lib/graphql_rails/errors/system_error.rb

```diff
@@ -3,6 +3,18 @@
 module GraphqlRails
   # base class which is returned in case something bad happens. Contains all error rendering tructure
   class SystemError < ExecutionError
+    attr_reader :error
+
+    def initialize(error)
+      super(error.message)
+
+      @error = error
+    end
+
+    def backtrace
+      error.backtrace
+    end
```

Comment:
✨  I would recommend use delegation instead of defining separate method.

```ruby
delegate :backtrace, to: :error
```

and put `error` attributte to private section

```
private

attr_reader :error
```

Reply:
@pucinsk, we will move the `error` back to the `public` as it might be handy during debugging